product_variant_csv_import/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3